services/wallet/README.md

```diff
@@ -0,0 +1,59 @@
+# Wallet
+
+### Add Relation Modules
+
+```
+~ ./qng --acctmode --modules=wallet --modules=account
+```
+
+### Create Or Import a account
+
+```
+~ ./qng --privnet -A=./ account import (privateKey path)
+or
+~ ./qng --privnet -A=./ account new
+```
+
+### Unlock account
+
+```
+~ ./cli.sh unlock 0 123456 60
+or 
+~
+curl -k -u test:test -k -X POST   https://127.0.0.1:38131/   -H 'content-type: application/json'   -d '{
+  "method":"wallet_unlock",
+  "version":"2.0",
+  "params":["0","123456",600],                                                                                                  "id":1
```

Comment:
if the input `0` means account index, I think there should a `list-accounts` command to list the accounts